### PR TITLE
Adding Raspberry Pi Pico to the Jupyter Notebook

### PR DESCRIPTION
**Pull request by jposada202020**
Adding the Raspberry Pi Pico to the list of boards available to test the Jupyter Notebook.

Board.py was change to include the Pico identification.
A new example was added using a picture from ADAFRUIT
readme.rst was modified to include Pico product reference

Worked with the Blinky example.
image

### CODE REVIEW

**Comment by jposada202020** [comment]
This was tested in Ubuntu 20.04

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (brentru) [inline .idea/.gitignore:1]: Why are these file's being ignored?

File: .idea/circuitpython_jupyter_kernel.iml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (brentru) [inline .idea/circuitpython_jupyter_kernel.iml:1]: Could you please delete the `.idea/` folder, you might've accidentally comitted it?

File: README.rst [hunk 3]
```diff
@@ -27,7 +27,7 @@ Status
 ------
 
 This project's status is experimental. It has been tested with CircuitPython (SAMD) boards and the
-Feather HUZZAH (ESP8266) with CircuitPython 3.x.
+Feather HUZZAH (ESP8266) with CircuitPython 3.x and Raspberry Pi Pico CircuitPython 6.x.
```
Comment (brentru) [inline README.rst:30]: Could you put this on a new line?

**Review by brentru — CHANGES_REQUESTED** [review]
@jposada202020 Thank you for contributing. An extra notebook for Pico is OK. Could you please make some small changes to clean this PR up a bit?

File: .idea/circuitpython_jupyter_kernel.iml (hunk 2 above)
Comment (jposada202020) [reply .idea/circuitpython_jupyter_kernel.iml:1]: Thanks for the revision, you are right, I might committed my Pycharm folder by accident. I will correct that.

File: README.rst (hunk 3 above)
Comment (jposada202020) [reply README.rst:30]: Yes will do,Thank you very much for the comments, I will review this tonight

**Comment by jposada202020** [comment]
@brentru Thanks for the revision, I made the changes mentioned, you were right, I remove the whole .idea directory, My IDE pushed it and I did not checked carefully. I modified the text trying to make more sense of the sentence. Let me know. Thanks.

**Review by brentru — APPROVED** [review]
Thank you for contributing, lgtm!

**Pull request merged**